File: Design and Programming Environments/Iteration 3/TicketManager/Networking/src/main/java/ObjectProtocol/AddTicketRequest.java
package ObjectProtocol;

import DTOPackage.TicketDTO;

public class AddTicketRequest implements Request{
    private TicketDTO ticketDTO;

    public TicketDTO getTicketDTO() {
        return ticketDTO;
    }

    public AddTicketRequest(TicketDTO ticketDTO) {
        this.ticketDTO = ticketDTO;
    }
}

File: Second year/Design and Programming Environments/TicketManagerCSharp/Networking/Requests/AddTicketRequest.cs
using Networking.DTOS;
using System;

namespace Networking.Requests
{
    [Serializable]
    public class AddTicketRequest : Request
    {
        private TicketDTO ticketDTO;

        public AddTicketRequest(TicketDTO ticketDTO)
        {
            this.ticketDTO = ticketDTO;
        }

        public TicketDTO TicketDTO { get => ticketDTO; set => ticketDTO = value; }
    }
}